### DISCUSSION TITLE
Difference between BatchedMetric and Metric

### DISCUSSION THREAD

**Discussion by smachar** [understandable-machine-intelligence-lab/Quantus#275] [Q&A]
From my understanding, please correct me if I am wrong, the ```BatchedMetric``` evaluate batch of instances over the batches generated by ```generate_batches``` method and for ```Metric``` it just iterates over the instances. If this true why there are those two options ? I mean why not using just the ```BatchedMetric``` which is equivalent to ```Metric``` when ```batch_size=1```.

It's kind of confusing as in every metric's method the input data argument is named ```x_batch``` and expected to be batched !!!

Thanks in advance for clarifying this :)

**Comment by annahedstroem** [comment]
Thanks @smachar! Unfortunately, I only saw this message now. But @aaarrti fixed this earlier this year (right now we only use the `Metric` class as suggested.
